viewer/src/main/webapp/viewer-html/components/Edit.js

```diff
@@ -1005,7 +1001,19 @@ Ext.define("viewer.components.Edit", {
             editable: !(attribute.hasOwnProperty('allowValueListOnly') && attribute.allowValueListOnly),
             labelClsExtra: this.editLblClass
         });
-
+        
+        //when device is mobile dont allow the touchend event. This will activate a field that is lying under the dropdown picker(ul)
+        if (viewer.components.MobileManager.isMobile()) {
+            input.on('focusenter', function () {
+                if (!input.eventInit) {
+                    document.getElementById(attribute.name + "-picker-listEl").addEventListener('touchend', function (e) {
```

Comment:
Check if the DOM element can be found. If future Ext updates change some names / elements this will return an error.

```suggestion
                    var pickerList = document.getElementById(attribute.name + "-picker-listEl");
                    if (!pickerList) {
                        return;
                    }
                    pickerList.addEventListener('touchend', function (e) {
```